src/main/resources/META-INF/primefaces-extensions.taglib.xml

```diff
@@ -6812,6 +6812,14 @@ See showcase for an example. Default null.]]>
             <required>false</required>
             <type>java.lang.String</type>
         </attribute>
+        <attribute>
+            <description>
+                <![CDATA[Keep the menu open when an item is clicked. Default value is false.]]>
+            </description>
+            <name>keepOpen</name>
+            <required>false</required>
+            <type>boolean</type>
```

Comment:
Change "boolean" to "java.lang.Boolean" and it will work!

Reply:
Both are used in PFE. I don't think it really matters in this case. Changed it though.